common/scripted_triggers/iisc_triggers.txt

```diff
@@ -91,22 +102,22 @@ iisc_should_build_builder = {
 		which = iisc_budget_builder
 		value >= @iisc_builder_cost
 	}
+	iisc_has_not_enough_builders = yes
+}
+
+# this = corp
+iisc_has_not_enough_builders = {
 	count_owned_ship = {
 		limit = {
 			is_ship_size = constructor
 		}
-		count < 1		# TODO change
+		count < 5
```

Comment:
Same as above - 5 constructors feels like a lot for a single corp. Not saying we need to change these now but it should definitely be something we keep an eye on and potentially change later.

Reply:
Reduced to 3. Might still be too much. But they should still construct enough stations for the player to notice them